test-prow-e2e.sh

```diff
@@ -13,6 +13,20 @@ function copyArtifacts {
   fi
 }
 
+function waitForAuthOperatorProgressing {
+  output=
+  action="started"
+  if [ "$1" == "False" ]; then
+    action="finished"
+  fi
+  until [ "$output" == "$1" ]; do
+    output=$(oc get co/authentication -o 'jsonpath={.status.conditions[?(@.type=="Progressing")].status}' 2>&1)
```

Comment:
Do we want to send stderr to stdout here? I'd think we'd just want

```suggestion
    output=$(oc get co/authentication -o 'jsonpath={.status.conditions[?(@.type=="Progressing")].status}')
```